tests/Oryx.BuildImage.Tests/PrepareEnvironmentCommandTest.cs

```diff
@@ -195,5 +195,42 @@ public void SkipsDetectionAndInstallsPlatformsFromFile()
                 },
                 result.GetDebugInfo());
         }
+
+        [Fact]
+        public void InstallsPlatformAtCustomInstallationRootDirectory()
+        {
+            // Arrange
+            var nodeVersion = "4.4.7";
+            var customDynamicInstallRootDir = "/foo/bar";
+            var expectedText =
+                $"Node path is: {customDynamicInstallRootDir}/{NodeConstants.PlatformName}/{nodeVersion}/bin";
+            var script = new ShellScriptBuilder()
+                .SetEnvironmentVariable("DYNAMIC_INSTALL_ROOT_DIR", customDynamicInstallRootDir)
+                .AddCommand(
+                $"oryx prep --skip-detection --platforms-and-versions " +
+                $"'{NodeConstants.PlatformName}={nodeVersion}'")
+                .AddDirectoryExistsCheck(
+                $"{customDynamicInstallRootDir}/{NodeConstants.PlatformName}/{nodeVersion}")
+                .AddCommand($"source benv node={nodeVersion}")
+                .AddCommand("nodePath=$(which node)")
+                .AddCommand("echo \"Node path is: $nodePath\"")
+                .ToString();
+
+            // Act
+            var result = _dockerCli.Run(new DockerRunArguments
+            {
+                ImageId = _imageHelper.GetGitHubActionsBuildImage(),
```

Comment:
yeah .. so I would like to add a test for dynamic install custom directory for full build image as well
1. where custom root dir is /opt/node or opt/nodejs with an existing node (e.g. 10)
2. with node 14 as it is been removed

Reply:
Good scenario. Will add it.